### PR TITLE
Home page completed with Home page tests. Footer component was touche…

### PR DESCRIPTION
**Pull request by TsunamiCode5**
…d just to remove the fixed position.

### CODE REVIEW

File: app/javascript/components/components/Footer.js [hunk 1]
```diff
@@ -1,10 +1,11 @@
 import React, { Component } from 'react'
 import {  } from 'reactstrap'
+import { ABSOLUTE_RESOURCE_PATH } from 'webpack/lib/ModuleFilenameHelpers'
```
Comment (elysemontano) [inline app/javascript/components/components/Footer.js]: What is this import for?

File: app/javascript/components/pages/Home.js [hunk 2]
```diff
@@ -1,10 +1,173 @@
...
+    if(window.pageYOffset > 300){
+      this.setState({is_visible: true})
+    } else {
+      this.setState({is_visible: false})
+    }
+  }
+
+  scrollToTop(){
+    window.scrollTo({top:0, behavior:"smooth"})
+  }
```
Comment (elysemontano) [inline app/javascript/components/pages/Home.js]: I would be careful when utilizing document and window directly inside React as this is directly manipulating the DOM which is not best practice.

File: app/javascript/components/pages/Home.js [hunk 3]
```diff
@@ -1,10 +1,173 @@
...
+              <UncontrolledCarousel
+                className="roundpics"
+                items={this.state.carouselPics}
+              />
+            </Col>
+          </Row>
+          <br />
+          <CardGroup className="homepage">
+            <Card>
+              {/* <CardImg alt="Card image cap" src="https://picsum.photos/318/180" top width="100%"/> */}
```
Comment (elysemontano) [inline app/javascript/components/pages/Home.js]: I see a lot of commented-out code throughout this section.  Please remove any code you are not using.

**Review by elysemontano — CHANGES_REQUESTED** [review]
Please see comments in Footer and Home

File: app/javascript/components/components/Footer.js (hunk 1 above)
Comment (TsunamiCode5) [reply app/javascript/components/components/Footer.js]: No I didn't put that lol but I will definitely remove. Good catch!

**Review by elysemontano — CHANGES_REQUESTED** [review]
I am still seeing the DOM being manipulated directly using window and document in Home.js.  This is not considered best practice in React usually.  I would recommend checking if there are other methods to create a scroll option that does not directly manipulate the DOM.

**Review by elysemontano — APPROVED** [review]

**Pull request merged**